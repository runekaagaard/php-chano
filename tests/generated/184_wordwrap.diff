001- this is a long
001+ Fatal error: Call to undefined method DtlIter::wordwrap() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/184_wordwrap.php on line 4
002- paragraph of
003- text that
004- really needs
005- to be wrapped
006- I'm afraid