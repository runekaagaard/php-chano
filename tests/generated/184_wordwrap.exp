this is a long
paragraph of
text that
really needs
to be wrapped
I'm afraid