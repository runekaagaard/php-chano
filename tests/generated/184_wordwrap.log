
---- EXPECTED OUTPUT
this is a long
paragraph of
text that
really needs
to be wrapped
I'm afraid
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::wordwrap() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/184_wordwrap.php on line 4
---- FAILED
