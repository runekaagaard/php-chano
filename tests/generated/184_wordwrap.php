<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'this is a long paragraph of text that really needs to be wrapped I\'m afraid')));
foreach ($items as $i) echo $i->input->wordwrap(14);
