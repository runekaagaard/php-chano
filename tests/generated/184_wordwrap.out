Fatal error: Call to undefined method DtlIter::wordwrap() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/184_wordwrap.php on line 4